lib/cmap/auth/sspi.js

```diff
@@ -43,12 +42,10 @@ class SSPI extends AuthProvider {
    * @method
    */
   auth(sendAuthCommand, connections, credentials, callback) {
-    if (kerberos == null) {
-      try {
-        kerberos = retrieveKerberos();
-      } catch (e) {
-        return callback(e, null);
-      }
+    try {
+      Kerberos.version;
```

Comment:
looks like you are covering the kerberos case here that I commented above. It's a little bit of a hack to test `Kerberos.version`, but maybe that's fine. try/catch doesn't deopt v8 anymore since turbofan I believe, but it would be more performant in general to be able to return this error in a callback without causing an error to be thrown.